hermes/file_gen_test.go

```diff
@@ -1,39 +1,38 @@
 package hermes
 
 import(
-	//"fmt"
 	"testing"
-	"math/rand"
 	"strconv"
-	"time"
-	//"crypto/sha1"
 )
 
 
 func TestChecksum(t *testing.T){
-	var global_contents string = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
-	var result_expected string = "_68f3caf439065824dcf75651c202e9f7c28ebf07"
+	global_contents := "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+	want := "_68f3caf439065824dcf75651c202e9f7c28ebf07" //expected checksum result
```

Comment:
Leave 2 spaces before an inline comment.
```suggestion
	want := "_68f3caf439065824dcf75651c202e9f7c28ebf07"  //expected checksum result
```